src/fetchers/tarball-fetcher.js

```diff
@@ -65,6 +65,13 @@ export default class TarballFetcher extends BaseFetcher {
 
     extractorStream
       .pipe(untarStream)
+      .on('entry', (entry) => {
+        if (entry.type == 'Directory' || entry.props.mode & 0o100) { // eslint-disable-line no-bitwise
+          entry.props.mode = 0o775;
```

Comment:
I am not sure the correct thing to do here. Obviously we want to normalize but maybe we might want to limit to just the current user account.